__tests__/fixtures/on-content/layouts/docs-page.jsx

```diff
@@ -0,0 +1,17 @@
+import { frontMatter as introData } from '../pages/docs/intro.mdx'
+import { frontMatter as advancedData } from '../pages/docs/advanced.mdx'
+
+export default frontMatter => {
+  return function docsPageLayout({ children }) {
+    return (
+      <>
+        <p>LAYOUT TEMPLATE</p>
+        <h1>{frontMatter.title}</h1>
+        <p>
+          Other docs: {introData.title}, {advancedData.title}
+        </p>
+        {children}
+      </>
```

Comment:
I don't think this content is needed since its not being tested - let's narrow the fixture down to only the things being tested!

Reply:
I think there might need to be some discussion here as the `onContent` function actually gets called within `processLayout()` in the `loader` .. so removing Layouts from this fixture broke my tests (:+1:) -- 

Do we want to support `onContent` for MDX that does not have a layout or did I perhaps overlook adding a 'default' Layout?

Reply:
@jescalan - :scissors: I cut this down here: https://github.com/hashicorp/next-mdx-enhanced/pull/26/commits/79e698572b4fb1897910d7e313926a154530b152